native/src/hooks/useUpdateParamsOnLanguageChange.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react'
+
+type UseOnLanguageChangeProps = {
+  languageCode: string
+  onLanguageChange: (newLanguageCode: string, previousLanguageCode: string) => void
+}
+
+const useOnLanguageChange = ({ languageCode, onLanguageChange }: UseOnLanguageChangeProps): void => {
```

Comment:
Should this really be named useOnLangugeChange?

What is the difference between this "useUpdateParamsOnLanguageChange" and "useOnLanguageChange"?
Which Params are updated? When do I use which hook?